frontend/src/pages/study-room-page/StudyRoomPage.tsx

```diff
@@ -0,0 +1,58 @@
+import { useState } from 'react';
+import { useParams } from 'react-router-dom';
+
+import { css } from '@emotion/react';
+
+import Wrapper from '@components/wrapper/Wrapper';
+
+import * as S from '@study-room-page/StudyRoomPage.style';
+import SideMenu from '@study-room-page/components/side-menu/SideMenu';
+
+export type Tab = { id: string; name: string; contents: React.ReactNode };
+
+export type Tabs = Array<Tab>;
+
+export type TabId = Tab['id'];
+
+const tabs: Tabs = [
+  { id: 'notice', name: '공지사항', contents: '공지사항입니다.' },
+  { id: 'material', name: '자료실', contents: '자료실입니다.' },
+  { id: 'review', name: '후기', contents: '후기입니다.' },
+];
+
+const StudyRoomPage: React.FC = () => {
+  const { studyId } = useParams();
```

Comment:
```suggestion
  const { studyId } = useParams() as { studyId: string };
```
이렇게 안해줘도 문제 없나요? 이전에는 다 붙여줬던것 같아서요

Reply:
이 변수를 아직 사용하지 않아서 문제가 안 생긴 것 같네요! 그래도 혹시 모르니 붙여놓을게요